<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bookAnalysisWindow</class>
 <widget class="QWidget" name="bookAnalysisWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1432</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图书管理系统 - 图书信息统计</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <selectedon>D:/pictures/avator.png</selectedon>
   </iconset>
  </property>
  <widget class="QLabel" name="bookCountLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>1411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>图书表中共有 0本书</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>1411</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="pressGroupBox">
      <property name="title">
       <string>出版社</string>
      </property>
      <widget class="QLabel" name="pressCountLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>681</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>共有 0个出版社</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="pressComboBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>671</width>
         <height>22</height>
        </rect>
       </property>
       <property name="currentText">
        <string/>
       </property>
      </widget>
      <widget class="QTableWidget" name="pressBookTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>681</width>
         <height>481</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>165</number>
       </attribute>
       <column>
        <property name="text">
         <string>ISBN</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>书名</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>作者名</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>价格</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="selectedPressCountLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>681</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>所选出版社共有 0本书</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="authorGroupBox">
      <property name="title">
       <string>作者</string>
      </property>
      <widget class="QLabel" name="authorCountLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>681</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>共有 0位作者</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="authorComboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>681</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableWidget" name="authorBookTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>681</width>
         <height>481</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>165</number>
       </attribute>
       <column>
        <property name="text">
         <string>ISBN</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>书名</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>价格</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>出版社</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="selectedAuthorCountLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>681</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>所选作者共有 0本书</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>670</y>
     <width>1411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回上一级</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
